src/components/Employees/index.tsx

```diff
@@ -0,0 +1,214 @@
+        }
+    }
+
+    function findFilters(filterArray: string[], target: string) {
+        const foundFilters: string[] = [];
+        filterArray.forEach((filter) => {
+            if (filter !== target) {
+                foundFilters.push(filter);
+            }
+        });
+        return foundFilters;
+    }
+
+    function applyFilters(filters: string[], target: string) {
+        searchParams.delete(target);
+        setSearchParams(searchParams);
+        filters.forEach((filter) => {
+            searchParams.append(target, filter);
+        });
+        setSearchParams(searchParams);
+    }
+
+    function removeFilter(target: string, keyParam: string) {
+        if (searchButtonExists(target)) {
+            const newSearchActiveButtons = searchActiveButtons.filter(function (
+                item
+            ) {
+                return item !== target;
+            });
+            setSearchActiveButtons(newSearchActiveButtons);
+            const search = searchParams.getAll(keyParam);
+            const foundFilters = findFilters(search, target);
+            applyFilters(foundFilters, keyParam);
+        }
+    }
+
+    function filterEmployees(
+        employeesToFilter: IEmployeeItem[],
+        keyParam: string
+    ) {
+        const search = searchParams.getAll(keyParam);
+        let filteredEmployees: IEmployeeItem[] = [];
+        if (search.length === 0) {
+            filteredEmployees = employeesToFilter;
+        } else {
+            employeesToFilter.forEach((employee) => {
+                search.forEach((filter) => {
+                    if (
+                        employee.job.toLowerCase() === filter.toLowerCase() ||
+                        employee.availability.toLowerCase() ===
+                            filter.toLowerCase()
+                    ) {
+                        if (!employeeExists(employee, filteredEmployees)) {
+                            filteredEmployees.push(employee);
+                        }
+                    }
+                });
+            });
+        }
+        return filteredEmployees;
+    }
+
+    return (
+        <div className={styles['employees']}>
+            <h1 className={styles['page-title']}>
+                {t('description.employees')}
+            </h1>
+            <div className={styles['input-search-icon-wrapper']}>
+                <input
+                    className={styles['input']}
+                    placeholder={t('description.search')}
+                    onChange={(event) => {
+                        const search = event.target.value;
+                        if (search) {
+                            searchParams.set('search', search);
+                            setSearchParams(searchParams);
+                        } else {
+                            searchParams.delete('search');
+                            setSearchParams(searchParams);
+                        }
+                    }}
```

Comment:
if event handler has multiple lines of code it should be extracted to a function